idea/build.gradle.kts

```diff
@@ -78,6 +78,7 @@ dependencies {
     compile(project(":idea:kotlin-gradle-tooling"))
     compile(project(":idea:line-indent-provider"))
     compile(project(":plugins:uast-kotlin"))
+    compile(project(":plugins:uast-kotlin-fir"))
```

Comment:
We should not add this dependency for FE1.0 plugin, let's add it to `:idea-fir` module instead